packages/salesforcedx-vscode-core/src/commands/functions/functionService.ts

```diff
@@ -207,6 +208,8 @@ export class FunctionService {
         if (functionExecution) {
           functionExecution.debugSession = undefined;
         }
+
+        TraceFlagsRemover.getInstance(undefined).removeNewTraceFlags();
```

Comment:
If this seems hokey, let me know.  I don't want to create another "getInstance" function, and TypeScript doesn't support function overloading.  (yes, you can sometimes simulate function overloading in TS but it's hacky and I don't care for it).

I think this is safe b/c:
1) I know that by the time we get here, `getInstance(this.connection)1 (in `createTraceFlag()`) has already been called.
2) I added guards in getInstance() to guard against this.